app/services/classroom/page.tsx

```diff
@@ -0,0 +1,169 @@
+                </li>
+              </ul>
+            </div>
+          </CardContent>
+        </Card>        
+      </section>
+
+      {/* Computational Notebooks */}
+      <section className="content-wrapper py-24 px-14 lg:px-36">
+        <SectionHeader title="Computational Notebooks" align="center" />
+
+        <Card className="w-full border-none shadow-none rounded-none">
+          <CardContent className="mx-auto flex items-center px-6">
+            <div>
+              <p className="text-black text-xl pb-8">
+                Computational Notebooks provide a convenient, cloud-hosted way to serve Jupyter Notebooks for multiple users. Notebooks are launched within a pre-configured computing environment so that users do not need to install any software packages. This set-up free environment is ideal for courses and workshops where instructors intend for students to begin coding with minimal obstacles. Jupyter’s flexibility allows instructors to pick the preferred language for a particular context, including Python, Julia, R and many more.
+              </p>
+            </div>
+            </CardContent>
+            </Card>
+
+            <Card className="w-full border-none shadow-none rounded-none">
+              <CardContent className="mx-auto flex items-center px-6">
+                <div className="text-black text-xl pb-8">
+                  <h1 className="text-3xl font-semibold pb-8">Google Colab</h1>
+                    <p >
+                      Google Colab comes bundled with most Python-specific software libraries, and it supports real-time collaboration. It integrates with Google Drive and should be sufficient for most classroom or workshop needs.
+                    </p>
+                    <Button variant="primary_filled" size="lg">Learn More</Button>
+                </div>
+              </CardContent>
+            </Card>
+
+            <Card className="w-full border-none shadow-none rounded-none">
+            <CardContent className="mx-auto flex items-center px-6">
+            <div>
+              <h1 className="text-3xl font-semibold pb-8">Brown JupyterLab</h1>
+              <p className="text-black text-xl">
+                For more advanced needs, Brown’s JupyterHub may be a good fit for your needs. If you are an instructor, CCV can provide access to JupyterHub for your class or workshop, and Digital Learning and Design (DLD) can assist with integrating computational assignments into curricula. The implementation is supported by Brown OIT; please follow the link below to request an instance for your class. OIT staff will respond to your request to begin the setup process. We ask that requests for JupyterHub be made at least two months in advance of expected course deployment.
+              </p>
+              <Button variant="primary_filled" size="lg">Learn More</Button>
+              <Button variant="primary_filled" size="lg">Request Help</Button>
```

Comment:
Rename to "Request a Hub" and link to `https://docs.google.com/forms/d/e/1FAIpQLSct9rFCxLhPIezHI-RYRyEuSnvHrPZLMuUSFRTriIyd_3TAfA/viewform?usp=sf_link`